app/main/views/sign_in.py

```diff
@@ -139,34 +140,22 @@ def verify_email(user, redirect_url):
 
 
 def _handle_e2e_tests(redirect_url):
-    try:
-        current_app.logger.warning("E2E TESTS ARE ENABLED.")
-        current_app.logger.warning(
-            "If you are getting a 404 on signin, comment out E2E vars in .env file!"
-        )
-        user = user_api_client.get_user_by_email(os.getenv("NOTIFY_E2E_TEST_EMAIL"))
-        activate_user(user["id"])
-        return redirect(
-            url_for(
-                "main.show_accounts_or_dashboard",
-                next="EMAIL_IS_OK",
-            )
-        )
-    except Exception as e:
-        stre = str(e)
-        stre = stre.replace(" ", "_")
-        # Trying to get a message back to playwright somehow since we can't see the admin logs
-        return redirect(url_for(f"https://{stre}"))
+    current_app.logger.warning("E2E TESTS ARE ENABLED.")
+    current_app.logger.warning(
+        "If you are getting a 404 on signin, comment out E2E vars in .env file!"
+    )
+    user = user_api_client.get_user_by_email(os.getenv("NOTIFY_E2E_TEST_EMAIL"))
+    activate_user(user["id"])
+
+    if redirect_url and is_safe_redirect_url(redirect_url):
+        return redirect(redirect_url)
+
+    return redirect(url_for("main.show_accounts_or_dashboard"))
```

Comment:
I think this whole section might need to be updated again as it is undoing the additional error handling that was recently put into place.

@terrazoon could you please take a look here and double check?